### ISSUE TITLE
Install an Ambassador bot

### ISSUE THREAD

**Issue by nolanlawson** [tootcafe/discussions#12]
This project is recent but it seems like a good idea: [ambassador](https://github.com/mbilokonsky/ambassador).

You should read the description to understand what it's doing, but basically it would promote the most fav'd toots from our instance and allow remote instances to follow it in order to seed their instances with interesting content. This is a proposed fix to the problem of followbots.

Anybody see any problem with this approach? If we want to install it, it looks like I'd have to be the one to do it since it runs server-side.

**Comment by Pajn** [comment]
I think that's a great idea!
Maybe we could also link it's profile from about or more page (continuing from #7) so that people can get some feeling of this instance before they join?

**Comment by nolanlawson** [comment]
Yup, I think that's another one of the goals of the project, so sounds like a great idea to link from the homepage.

**Comment by zcdunn** [comment]
I really like the idea of ambassador bots. New users and users on other instances can see the best of :trumpet: :coffee: I also would like to see other instances installing one, so it becomes common for users to check an instance's ambassador handle for good stuff.

**Comment by nolanlawson** [comment]
I'm a bit concerned that only one instance seems to actually be using an ambassador bot. Also it's third-party code running directly against the database. Gonna hold off for now. I hope the core Mastodon project adds local timelines.
